src/webgpu/util/texture/texel_data.ts

```diff
@@ -122,6 +122,21 @@ export function makeClampToRange(format: EncodableTextureFormat): ComponentMapFn
   return applyEach(x => clamp(x, repr.numericRange!), repr.componentOrder);
 }
 
+const smallComponentDataViews = new Map();
```

Comment:
perhaps have a MAINTENANCE_TODO about exposing this map somehow to the test fixture so we can GC its contents at the end of a test group.

Then, we could potentially consider storing > 32-byte buffers in it as well
(not sure how helpful that would be)

Reply:
Added a comment.